### ISSUE TITLE
Explain how to setup better

### ISSUE THREAD

**Issue by ZyvoksCode** [GayarraFrost/DiscordTokenGrabber#1]
what white window and what link, if you could a small tutorial link, appreciated

**Comment by GayarraFrost** [comment]
Yep, I can do that if you want give me your Discord i will send you a video how to setup that when i've got some time

**Comment by ZyvoksCode** [comment]
> Yep, I can do that if you want give me your Discord i will send you a video how to setup that when i've got some time

Alr my discord is Trex ‘ 2.0#1111

**Comment by GayarraFrost** [comment]
I Can't add you lmao, can you activate ur friend request ?

**Comment by Projectvulse** [comment]
bro the thing is it's not working for me can you dm me via discord GlobalVulse#0001
